Add Fysetc V2.1 / MKS V3 / BTT V1 Mini 12864 support for BTT SKR Mini E3

minteyay opened the pull request:
### Description

Added the pin configuration for using a BIGTREETECH MINI12864 / MKS MINI12864 V3 / Fysetc Mini 12864 V2.1 LCD board with a BTT SKR Mini E3.

### Requirements

Requires one of the mentioned LCD boards and the specific mainboard.

### Benefits

Enables the use of the LCD board (without SD card support) with the specific mainboard, with the use of an adapter cable (documented in comments).

minteyay committed:
Update pins_BTT_SKR_MINI_E3_common.h

Add Fysetc Mini 12864 V2.1 / BTT Mini 12864 V1 support for BTT SKR Mini E3

thisiskeithb commented on Marlin/src/pins/stm32f1/pins_BTT_SKR_MINI_E3_common.h:
```diff
@@ -235,6 +235,70 @@
 
     #endif
+  
+  #elif EITHER(FYSETC_MINI_12864_2_1, BTT_MINI_12864_V1)
```
You only need to check for `FYSETC_MINI_12864_2_1` and this can apply to the MKS Mini 12864 V3 lcd as well: https://github.com/MarlinFirmware/Marlin/blob/22b99a2e382034f951568dda8d2c70af06ddc47f/Marlin/src/inc/Conditionals_LCD.h#L34-L37

minteyay replied:
Ah, thank you! I think VSCode didn't recognise it with just `FYSETC_MINI_12864_2_1`, but that makes sense given that `BTT_MINI_12864_V1` is defined as an alias. Going to make the changes soon :>

thisiskeithb replied:
VSCode tends to get confused due to all the macros used in Marlin.

minteyay committed:
Check only for FYSETC_MINI_12864_2_1

minteyay committed:
Add MKS_MINI_12864_V3 to descriptions

minteyay committed:
Add displays to list of supported LCDs

minteyay commented:
Okay, I changed the check for the simpler one, added the MKS Mini V3 into the descriptions, and also added the three displays to the error message telling you which displays are supported.
